public/apps/configuration/panels/role-edit/role-edit.tsx

```diff
@@ -180,12 +187,19 @@ export function RoleEdit(props: RoleEditDeps) {
 
   const indexWisePermissionOptions = [
     {
-      label: 'Permission groups',
+      label: 'Index permission groups',
       options: actionGroups
         .filter((actionGroup) => actionGroup[1].type === 'index')
         .map((actionGroup) => actionGroup[0])
         .map(stringToComboBoxOption),
     },
+    {
```

Comment:
Are there permission groups that have combined cluster and index permissions?

Reply:
Just updated the description. There are also "kibana", "all", and undefined (custom) types.